public/resolve.js

```diff
@@ -0,0 +1,42 @@
+const { rpc, u } = require('@cityofzion/neon-js');
+
+// TODO: Configurable network settings and script hash
```

Comment:
PR Looks good to me besides this TODO, but it can be merged (after the CI Build is fixed) 
We can add this in later

Reply:
Fixed the build, merging!